docs/source/guides/contributor/chapters/plugins.rst

```diff
@@ -27,8 +27,65 @@ world output in the console.
           :py:data:`avocado.core.output.LOG_JOB`, which can be used to
           output to job log files when running a job.
 
+Registering configuration options (settings)
+============================================
+
+It is very common for a plugin to allow users to configure something. A plugin
+might change its behavior depending on a specific configuration option.
+
+Usually, those configurations came basically from configuration files and
+sometimes from the command-line argument options. Like in most UNIX-like tools,
+command-line options will override values defined inside the configuration
+files.
+
+You, as a plugin writer, don’t need to handle this configuration by yourself.
+Avocado provides a common API that can be used by plugins in order to register
+options and get values.
+
+If you would like to give to your users some options to use, you can register
+an option using the :meth:`.Settings.register_option()` method during your
+plugin configuration stage. And use the config dictionary that is provided to
+you with the options already parsed.
+
+Let’s take our Hello World example and change the message based on a “message”
+option:
+
+.. literalinclude:: ../../../../../examples/plugins/cli-cmd/hello/hello_option.py
+
+This registration will register a “configuration namespace (“hello.message”)
```

Comment:
there is a missing double quote close here or not necessary before the `configuration`.